src/bundle/Resources/public/css/structure.css

```diff
@@ -0,0 +1,36 @@
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+}
+
+ez-platform-ui-app {
+    display: flex;
+    flex: 1 0 100%;
+    flex-direction: column;
+}
+
+ez-navigation-hub {
+    flex: 0 0;
+}
+
+main {
+    flex: 1 0;
+    display: flex;
+}
+
+#discovery {
+    flex: 0 0 230px;
```

Comment:
OK, I'll adjust the width to 115px by default. For the rest (especially `position: fixed`), it really depends on the expected behavior while scrolling.